datafusion/physical-expr/src/datetime_expressions.rs

```diff
@@ -424,6 +425,84 @@ fn to_timestamp_impl<T: ArrowTimestampType + ScalarType<i64>>(
     }
 }
 
+/// # Examples
+///
+/// ```
+/// use std::sync::Arc;
+
+/// use datafusion::arrow::array::StringArray;
+/// use datafusion::arrow::datatypes::{DataType, Field, Schema};
+/// use datafusion::arrow::record_batch::RecordBatch;
+/// use datafusion::error::Result;
+/// use datafusion::prelude::*;
```

Comment:
Unfortunately we'll have to add an ignore for this because of https://github.com/apache/arrow-datafusion/issues/9277 See https://github.com/apache/arrow-datafusion/pull/9279/files#diff-67ae8808785b2e651767d7ff67dd7c53be04ca098857b52c82ed19927e071cdaR514 for an example. I believe once the functions are move to the new dataframe-functions crate we should be able to re-enable.